src/viam/sdk/components/camera/camera.cpp

```diff
@@ -7,9 +7,15 @@
 #include <viam/api/component/camera/v1/camera.grpc.pb.h>
 #include <viam/api/component/camera/v1/camera.pb.h>
 
+#include <viam/sdk/common/exception.hpp>
 #include <viam/sdk/common/utils.hpp>
 #include <viam/sdk/resource/resource.hpp>
 
+#ifndef htonll
+#define htonll(b) \
+    ((1 == htonl(1)) ? (b) : ((uint64_t)htonl((b)&0xFFFFFFFF) << 32) | htonl((b) >> 32))
```

Comment:
Ahh I see where the `arpa` include comes into play here. If we must `include` it I'd prefer it to be in the `cpp` file rather than the `hpp`. But, it looks like we can do `endian` conversions with [boost](https://www.boost.org/doc/libs/1_84_0/libs/endian/doc/html/endian.html#conversion), which we already are relying on. can we swap to that?

Reply:
Good call. I didn't realize boost could do that. Might be best to refactor the Realsense to use that too... diff conversation though